components/Modal.js

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useRef, useState } from "react";
+import ReactDOM from "react-dom";
+import styled from "styled-components";
+
+const Modal = ({ show, onClose, children, title }) => {
+  const [isBrowser, setIsBrowser] = useState(false);
```

Comment:
I don't know what's the reason to do that, because I can't see when the browser could be false

Reply:
i tried to used modal but i changed method of push a new reviews, i deleted the modal and all of his methods and components